packages/terra-responsive-element/src/ResponsiveElement.jsx

```diff
@@ -111,7 +111,12 @@ class ResponsiveElement extends React.Component {
 
   render() {
     if (this.state.element) {
-      return this.props[this.state.element] || null;
+      const Tag = this.props[this.state.element] || null;
+      if (Tag) {
+        return <Tag {...this.props} />;
```

Comment:
Can you write a few tests that add custom properties into the component and verify they are present upon render?

Reply:
Oh duh, of course. That makes a lot more sense than what i'm doing

Reply:
This would also need to merge existing classNames with custom classNames similar to how many of our components are currently doing this:

https://github.com/cerner/terra-core/blob/master/packages/terra-card/src/Card.jsx#L44